ecs-init/docker/docker.go

```diff
@@ -23,7 +23,6 @@ import (
 
 	"github.com/aws/amazon-ecs-init/ecs-init/backoff"
 	"github.com/aws/amazon-ecs-init/ecs-init/config"
-	"github.com/aws/amazon-ecs-init/ecs-init/gpu"
```

Comment:
Just an idea: If this change is going to be here for a while, we could create a .patch file and put it in the focal packaging. That way, this change is only applied when we build Ubuntu.